store-hints.py

```diff
@@ -6,9 +6,13 @@
 
 con = sqlite3.connect('word2vec.db')
 cur = con.cursor()
-cur.execute("""create table if not exists nearby (word text, neighbor text, similarity float, percentile integer)""")
+cur.execute("""drop table similarity_range;""")
```

Comment:
Will this error if the table does not exist?

Reply:
Yes, fixed.